Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -461,7 +470,7 @@ function Get-AzRmResourceGroup
 
         $proxyUri = Get-ProxyUri $uri
 
-        if (($proxyUri -eq $null) -or ($proxyUri.AbsoluteUri -eq $null) -or ($proxyUri.AbsoluteUri -eq $uri))
+        if (($proxyUri -eq $null) -or ($proxyUri.AbsoluteUri -eq $null) -or ($uri -like "*$($proxyUri.Host)*"))
```

Comment:
Aren't SQL Azure tasks using proxy. Do they have any methods that needs update as well?